src/Main.java

```diff
@@ -1,5 +1,93 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+            //ЗАДАЧА1
+        byte b = 100;
+        short sh = 300;
+        int i = 40000;
+        long l = 100000000l;
+        float f = 100.112f;
+        double d = 102.9999;
+        System.out.println("Значение переменной b с типом byte равно " + b);
+        System.out.println("Значение переменной sh с типом short равно " + sh);
+        System.out.println("Значение переменной i с типом int равно " + i);
+        System.out.println("Значение переменной l с типом long равно " + l);
+        System.out.println("Значение переменной f с типом float равно " + f);
+        System.out.println("Значение переменной b с типом double равно " + d);
+        //ЗАДАЧА2
+        float fl = 27.12f;
+        long k = 987678965549l;
+        double digits = 2.786;
+        short ssshhhh = 569;
+        int n = -159;
+        int iAmArtem = 27897;
+        byte botBob = 67;
+        //ЗАДАЧА3
+        byte lp = 23;
+        byte an = 27;
+        byte ea = 30;
+        short vsegoUchenikov = (short) (lp + an + ea);
+        short vsegoListov = 480;
+        double listovNaUchenika = vsegoListov/vsegoUchenikov;
+        System.out.println("...");
+        System.out.println("На каждого ученика рассчитано "+ listovNaUchenika + " листов бумаги");
+        ///ЗАДАЧА4
+        System.out.println("...");
+        byte bottlesEveryTwoMinutes = 16;
+        byte bottlesRepMinute = 16/2;
```

Comment:
```suggestion
        byte bottlesRepMinute = bottlesEveryTwoMinutes/2;
```
надо использовать переменную